widgets/google-map/fields/js/location-field.js

```diff
@@ -0,0 +1,177 @@
+/* global jQuery, sowbForms */
```

Comment:
Yeah, it's specific to the maps widget, because it requires the API key field somewhere in the form, which it uses to load the Maps API. This is a point which needs more consideration. Currently it's necessary for users to either have added their API key to global settings for the maps widget, or set the API key field in the widget form and save the page, before they can use this.